tool/libclang_config.yaml

```diff
@@ -76,6 +76,7 @@ functions:
     - clang_getPointeeType
     - clang_getCanonicalType
     - clang_Type_getNamedType
+    - clang_Type_getAlignOf
```

Comment:
Every time you use a new part of the clang API by using ffigen itself it makes me happy! 😄

Reply:
Haha, me too 😆